rust/lance-file/src/v2/writer.rs

```diff
@@ -337,12 +337,6 @@ impl FileWriter {
             .collect::<FuturesUnordered<_>>();
         self.write_pages(encoding_tasks).await?;
 
-        // No data, so don't create a file
-        if self.rows_written == 0 {
```

Comment:
why do we delete this ?

Reply:
for IVF_FLAT, the FLAT index file is empty, but we still need to store some metadata (IVF related)